### PR TITLE
VZ-8117-pod sec for velero

### PR DESCRIPTION
**Pull request by theabmitra**
- Enable pod sec as per https://kubernetes.io/docs/concepts/security/pod-security-standards/ for velero and restic pods. 
- Update pod sec tests for velero
- Remove prometheus dependency from velero install

### CODE REVIEW

**Comment by michael-cico** [comment]
looks good to me, although we probably need to add the `verrazzano-backup` namespace to the `verify-install/security` test suite.   Looks like that runs in the backup test pipelines, so we should get coverage there.

File: platform-operator/helm_config/overrides/velero-override-static-values.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022, Oracle and/or its affiliates.
+# Copyright (c) 2023, Oracle and/or its affiliates.
```
Comment (mgianatagh) [inline platform-operator/helm_config/overrides/velero-override-static-values.yaml:1]: you cannot remove the previous copyright date.

The change should be "2022, 2023"

Comment (theabmitra) [reply platform-operator/helm_config/overrides/velero-override-static-values.yaml:1]: ill try this again, but this seemed to fail locally

File: platform-operator/controllers/verrazzano/component/velero/velero_operator.go [hunk 2]
```diff
@@ -36,6 +36,9 @@ func AppendOverrides(compContext spi.ComponentContext, _ string, _ string, _ str
 		{Key: "initContainers[0].imagePullPolicy", Value: "IfNotPresent"},
 		{Key: "initContainers[0].volumeMounts[0].name", Value: "plugins"},
 		{Key: "initContainers[0].volumeMounts[0].mountPath", Value: "/target"},
+		{Key: "initContainers[0].securityContext.allowPrivilegeEscalation", Value: "false"},
```
Comment (mgianatagh) [inline platform-operator/controllers/verrazzano/component/velero/velero_operator.go:39]: copyright needs to be updated to "2022, 2023,"

Comment (michael-cico) [inline platform-operator/controllers/verrazzano/component/velero/velero_operator.go:39]: this needs a copyright update also right?

File: platform-operator/controllers/verrazzano/component/velero/velero_operator_component.go [hunk 3]
```diff
@@ -9,15 +9,13 @@ import (
 	"path/filepath"
 
 	"github.com/verrazzano/verrazzano/pkg/k8s/ready"
-	"github.com/verrazzano/verrazzano/pkg/vzcr"
-	"github.com/verrazzano/verrazzano/platform-operator/controllers/verrazzano/component/networkpolicies"
-	prometheusOperator "github.com/verrazzano/verrazzano/platform-operator/controllers/verrazzano/component/prometheus/operator"
-
 	"github.com/verrazzano/verrazzano/pkg/k8s/resource"
+	"github.com/verrazzano/verrazzano/pkg/vzcr"
```
Comment (mgianatagh) [inline platform-operator/controllers/verrazzano/component/velero/velero_operator_component.go:13]: copyright needs to be updated to "2022, 2023"

Comment (michael-cico) [inline platform-operator/controllers/verrazzano/component/velero/velero_operator_component.go:13]: this needs a copyright update also right?

**Review by michael-cico — APPROVED** [review]

File: platform-operator/controllers/verrazzano/component/velero/velero_operator.go (hunk 2 above)
Comment (theabmitra) [reply platform-operator/controllers/verrazzano/component/velero/velero_operator.go:39]: done

Comment (theabmitra) [reply platform-operator/controllers/verrazzano/component/velero/velero_operator.go:39]: done

File: platform-operator/controllers/verrazzano/component/velero/velero_operator_component.go (hunk 3 above)
Comment (theabmitra) [reply platform-operator/controllers/verrazzano/component/velero/velero_operator_component.go:13]: done

Comment (theabmitra) [reply platform-operator/controllers/verrazzano/component/velero/velero_operator_component.go:13]: done

**Pull request merged**